sdk/cosmosdb/cosmos/src/utils/batch.ts

```diff
@@ -168,6 +168,11 @@ export function decorateOperation(
   ) {
     const pk = extractPartitionKey(operation.resourceBody, definition);
     return { ...operation, partitionKey: JSON.stringify(pk) } as Operation;
+  } else if (
+    operation.operationType === BulkOperationType.Read ||
+    operation.operationType === BulkOperationType.Delete
+  ) {
+    return { ...operation, partitionKey: "[{}]" };
```

Comment:
Add a test for this case

Reply:
the modified test case is this case for Delete